include/ofi_util.h

```diff
@@ -327,22 +326,6 @@ ofi_ep_fid_bind(struct fid *ep_fid, struct fid *bfid, uint64_t flags)
 			   bfid, flags);
 }
 
-static inline void ofi_ep_lock_acquire(struct util_ep *ep)
-{
-	ep->lock_acquire(&ep->lock);
```

Comment:
can't we call ofi_genlock here?

Reply:
We can. I just thought it was silly for an inline function to just be one line that calls another inline function.